### DISCUSSION TITLE
Question: how do I set the version for a custom build?

### DISCUSSION THREAD

**Discussion by giovtorres** [trinodb/trino#18963] [Q&A] [answered]
I'm trying to build a local fork of Trino and can't find where to set the version so that it shows up correctly in the UI.

For example, I just updated from 403 to 425 and the version the UI tells me is `403-3843-GCC60D2C-DIRTY`. I would expect it to have `425` in the version, and it would seem I'm not updating this value somewhere.

Where does this get set for custom builds? Thanks.

**Comment by hashhar** [comment, marked as answer]
The version comes using the maven-gitcommitid plugin which uses `git describe` output. So check if your repo has all the tags and the history isn't diverged. So check what `git describe` returns for you.

You can tag the commit you are building with something and that will get used as the version.

**Comment by giovtorres** [comment]
Thank you! Tagging the commit worked!

**Reply by amir-bashir** [reply]
Hi, 
I am facing same problem for 427. My cluster UI shows me 427-DIRTY. However **git describe** show 427 only. Can you suggest what I should do?
